CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+
+## 🤔 Choosing an issue
+
+Below is a link to a list of issues that encompass a variety of different technologies, from Airflow to Django to GitHub Actions to Vue.js. You're welcome to take on any of the issues. Most require development environment set up, but some do not. Expect development environment set up to take 1 to 2 hours depending on experience and how much software you've already got installed. If you have git, a text editor/IDE, Python or Node and Docker set up, things will go a little more smoothly. If not, that's totally fine! We're here to help and our repositories all have README files that should include helpful instructions for self-guided environment set up.
+
+Once you've picked an issue to work on, please leave a comment saying so on the issue and include `@openverse-maintainers` in the comment. That way one of us will get a notification and can assign the issue to you.
+
+## 🤓 Initial set up
+
+### 🪟 Note for Windows users
+
+Windows Subsystem for Linux can be a much more versatile and familiar environment for software development on Windows. Everything from installing `git` and other dependencies to using command line tools that will be familiar to the wider community of software developers are more likely to be easier under WSL. While some parts of some Openverse projects may be able to be developed under native Windows, you will have a much smoother time with WSL as our command runners (`just` and `pnpm`) assume a unix-type environment (Linux and macOS). We would like to support native Windows development, but we do not currently have any active contributors who use Windows on a daily basis and do not have the time to spend supporting it. Likewise, many other OSS projects will not be configured to allow easy local development under Windows. WSL can help there as well.
+
+Installation instructions for WSL on Windows 10 and 11 can be found [here on Microsoft's official documentation website](https://docs.microsoft.com/en-us/windows/wsl/install).
+
+Of course, if you would like to contribute to Openverse by making our projects work natively on Windows, PRs for this are always welcome.
+
+### ✍️ General set up
+
+Skip any steps you've already completed on your own. This is meant as an exhaustive list for brand-new devs who might not have these tools yet.
+
+1. Install `git`
+    * Most Linux distributions and macOS will already have this installed. Open up your terminal app and type `which git`. If the response is anything other than `git not found` then you've already got it installed.
+    * If you need to install it, follow the official instructions here: https://git-scm.com/downloads
+2. Install a text editor.
+    * [PyCascades, an event we've attended previously, has a great list of common editors](https://mentored-sprint-for-diverse-beginners.readthedocs.io/01_contributor_resources.html#text-editors). VSCode in particular is very popular for Python and JavaScript developers alike.
```

Comment:
This looks like it might actually be documentation for PyCon US 2020 😮

Reply:
We might want to leave the link out as well because any link to an editor review might age very soon.